pyanaconda/modules/payloads/source/repo_files/repo_files.py

```diff
@@ -43,6 +43,14 @@ def description(self):
         """Get description of this source."""
         return _("Closest mirror")
 
+    @property
+    def network_required(self):
+        """Does the source require a network?
+
+        :return: True or False
+        """
+        return True
```

Comment:
Is this true in the mathematical sense, or just the best approximation? I could see repo files with only file://, but maybe these are special cases handled by other sources?

Reply:
It is the best approximation. This code will replace the implementation of the property `needs_network` from `DNFPayload` and we assume there that closest mirror requires network. See: https://github.com/rhinstaller/anaconda/blob/master/pyanaconda/payload/dnf/payload.py#L158

Reply:
Sorry, here I'm stupid again - of course that's how the code worked previously. Thank you for the explanation!